### DISCUSSION TITLE
Wheather ATS can be installed on a windows platform

### DISCUSSION THREAD

**Discussion by TJUCoastlabRGQ** [amanzi/amanzi#839] [Q&A]
Hi everyone! In the install guide, the mentioned platforms are ubuntu and Mac, I wonder wheather ATS can be installed on Windows platform, thx

**Comment by fmyuan** [comment]
Yes. Windows OS now has a linux, called WSL, Windows Subsystem for Linux, which can be activated with Ubuntu or other WSL supported linux. May check it out in Windows website on how to setup.
